### ISSUE TITLE
Severe lag when in Emain, Dunby, new Festvia area

### ISSUE THREAD

**Issue by Theoneyouknow** [cursey/kanan#139]
This has been going on for the past month now. When I go into these areas ^ Dunby, emain, or Festvia, I get a large lag. I never had this problem before but I am very unsure as to what is the cause.

**Comment by ghost** [comment]
Do you get this lag when running mabi without kanan?

**Comment by Theoneyouknow** [comment]
Nope! No lag when running clean client. When im using Kanan and im moving my mouse around such as the buildings view, EXTREME lag occurs. Im not sure why this is happening or If im the only one ?

**Comment by Aahzmandius** [comment]
I'll guess it's because you have RenderDistance.js enabled, increases your render range, which can be too much for your computer to handle if you have the camera facing certain angles with too much geometry on screen.

**Comment by cursey** [comment]
Yes, it most likely is because of RenderDistance.js, you can either disable that mod or lower the `range` value in `config.toml` under the `[RenderDistance]` entry.
